mllib-common/src/main/scala/org/apache/spark/ml/attribute/attributes.scala

```diff
@@ -339,7 +338,7 @@ class NominalAttribute private[ml] (
   override def isNominal: Boolean = true
 
   private lazy val valueToIndex: Map[String, Int] = {
-    values.map(Utils.toMapWithIndex(_)).getOrElse(Map.empty)
+    values.map(_.zipWithIndex.toMap).getOrElse(Map.empty)
```

Comment:
here partially revert https://github.com/apache/spark/commit/9cae423075145d3dd81d53f4b82d4f2af6fe7c15 to avoid directly depending on spark-core